### ISSUE TITLE
Feature request: add OS cronjob support (callbacks)

### ISSUE THREAD

**Issue by patricksavalle** [tommcfarlin/WordPress-Plugin-Boilerplate#258]
Hi Tomm, would it be an idea to add support for external cronjobs? The Wordpress cronjobs are not very good (add load, queries, not reliable on low traffic moments), in our project we set up Linux cronjobs that call 'secret' and .htaccess protected URL's. 

It could be an idea to create a nice placeholder / template for this.

**Comment by mAAdhaTTah** [comment]
While Tom may think different, I think that's probably outside the scope of this boilerplate. Most plugins don't need cronjob support. Maybe worth putting in a Gist for a future "Resources" section though.

**Comment by patricksavalle** [comment]
Agree. But it would be a big overhead too.

**Comment by tommcfarlin** [comment]
@patricksavalle yeah - incorporating this into the Boilerplate would be a bit outside the scope; however, once I finish the initial round of documentation that I'm working on, we can look at how to incorporate this using a couple of examples.
